src/components/sections/communities/_partials/Section.tsx

```diff
@@ -35,7 +35,7 @@ interface SectionProps {
  */
 export default function Section({ title, titleBold = true, subtitle, id, hideSubtitleOnMobile = false, children, className }: SectionProps): ReactElement {
   return (
-    <div id={id} className={`text-xl md:text-.5xl px-0 py-6 ${className}`}>
+    <div data-testid="sectionId" id={id} className={`text-xl md:text-.5xl px-0 py-6 ${className}`}>
```

Comment:
can you keep the consistency and add an argument for this testid
```suggestion
    <div data-testid="sectionId" id={id} className={`text-xl md:text-.5xl px-0 py-6 ${className}`}>
```